openbb_terminal/stocks/comparison_analysis/ca_controller.py

```diff
@@ -174,9 +165,7 @@ def print_help(self):
         mt.add_param("_ticker", self.ticker)
         mt.add_raw("\n")
         mt.add_cmd("tsne", self.ticker)
-        mt.add_cmd("getpoly", self.ticker)
-        mt.add_cmd("getfinnhub", self.ticker)
-        mt.add_cmd("getfinviz", self.ticker)
+        mt.add_cmd("get", self.ticker)
```

Comment:
Can we get rid of everything above and basically have this command called `similar`?

The `tsne` can be included here with the source being `YahooFinance` - where we just use a ML algorithm to get the similar ones.

The comands above are no longer necessary because then this command can be used as 
```
similar AAPL
similar TSLA --source Polygon
```